pkg/radar/threat.go

```diff
@@ -0,0 +1,49 @@
+package radar
+
+import (
+	"github.com/dharmab/skyeye/pkg/bearings"
+	"github.com/dharmab/skyeye/pkg/brevity"
+	"github.com/dharmab/skyeye/pkg/coalitions"
+	"github.com/martinlindhe/unit"
+	"github.com/paulmach/orb/geo"
+)
+
+func (s *scope) Threats(coalition coalitions.Coalition) map[uint32][]brevity.Group {
```

Comment:
Should probably invert the returned data structure - map groups to the coalition units they threaten, instead of units to groups. That'll make it easier to minify the brevity. If a group threatens two or more units, use bullseye instead of BRAA.